mem/inmem_test.go

```diff
@@ -0,0 +1,26 @@
+package mem
+
+import (
+	"testing"
+
+	pstore "github.com/libp2p/go-libp2p-peerstore"
+	"github.com/libp2p/go-libp2p-peerstore/test"
+)
+
+func TestInMemoryPeerstore(t *testing.T) {
```

Comment:
Implementation-specific tests call out to the suite, providing a factory function to create their peerstore/addressbook.